benchmark/DriverBench/SingleDocBench.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace MongoDB\Benchmark\DriverBench;
+
+use Generator;
+use MongoDB\Benchmark\BaseBench;
+use PhpBench\Attributes as Bench;
+
+use function array_map;
+use function range;
+
+/** @see https://github.com/mongodb/specifications/blob/ddfc8b583d49aaf8c4c19fa01255afb66b36b92e/source/benchmarking/benchmarking.rst#single-doc-benchmarks */
+#[Bench\BeforeMethods('prepareDatabase')]
+final class SingleDocBench extends BaseBench
+{
+    public function prepareDatabase(): void
+    {
+        self::getCollection()->drop();
```

Comment:
> Drop the 'perftest' database.

I don't think the database name matter. It's `phplib_test`.

https://github.com/mongodb/mongo-php-library/blob/758984085c6f516df119b98eb2613e92516e1a18/benchmark/BaseBench.php#L39-L42

And the collection is `perftest`, not `corpus`.

https://github.com/mongodb/mongo-php-library/blob/758984085c6f516df119b98eb2613e92516e1a18/benchmark/BaseBench.php#L27-L32